crosscompile.bash

```diff
@@ -8,7 +8,7 @@ VERSION="`git describe --abbrev=0 --tags --exact-match || git rev-parse --short
 BUILD_DATE="`date -u +%Y%m%d%.%H%M%S`"
 export VERSION_STRING="$VERSION ($BUILD_DATE)"
 LOGGLY_TOKEN="469973d5-6eaf-445a-be71-cf27141316a1"
-LDFLAGS="-w -X main.version $VERSION -X main.buildDate $BUILD_DATE -X main.logglyToken LOGGLY_TOKEN"
+LDFLAGS="-w -X main.version $VERSION -X main.buildDate $BUILD_DATE -X github.com/getlantern/flashlight/logging.logglyToken $LOGGLY_TOKEN"
```

Comment:
This never worked, so nothing ever actually got logged to Loggly.  I fixed it to embed the token correctly and verified that we're getting data to Loggly now.